dallinger/pytest_dallinger.py

```diff
@@ -418,6 +419,7 @@ def debug_experiment(request, env, clear_workers):
             p.expect_exact(u"Local Heroku process terminated", timeout=timeout)
     finally:
         try:
+            print(p.read(1e6))
```

Comment:
A comment explaining what's happening here and why would be helpful, I think.

Reply:
Good idea, I've updated the code to make it more descriptive. I grappled with this for a while, unfortunately `pexpect` seems to be a bit buggy for the kind of behaviour we want (streaming all the relevant logs to pytest). My solution seems to work though!